hack/lib/util.sh

```diff
@@ -457,7 +457,7 @@ kube::util::godep_restored() {
 kube::util::ensure_clean_working_dir() {
   if ! git diff --exit-code; then
     echo -e "\nUnexpected dirty working directory."
-    exit 1
+    return 1
```

Comment:
It seems to be pretty mixed in utils.sh: some use exit, some use return. Exit is fine for me, but then we have to copy the git-diff command out of the function.